interpreter/attributes.go

```diff
@@ -96,6 +96,9 @@ type Attribute interface {
 
 	// Resolve returns the value of the Attribute given the current Activation.
 	Resolve(Activation) (interface{}, error)
+
+	// RuntimeCost returns the runtime cost of the Attribute given the current Activation and EvalState.
+	RuntimeCost(ctx Activation, evalState EvalState) uint64
```

Comment:
Does the `RuntimeCost` need the `ctx Activation`? It doesn't seem to be used.

Reply:
All the RuntimeCost interface calls have been dropped, so this is obselete.